.github/workflows/tests.yml

```diff
@@ -1,5 +1,10 @@
 name: Tests
 
+# The wheel package of jpy is not reliable and causes
+# run time errors, Forcing to build from source
+env:
+  PIP_NO_BINARY: jpy
```

Comment:
I think there is no need to setup the envvar, https://github.com/benthomasson/ansible-events/pull/144#issuecomment-1226043523

Reply:
@Alex-Izquierdo the jpy package comes in source and wheel distribution. The wheel distribution has a shared library for Java Native Interface which loads the share library for jvm. That has some issues which the jpy team is looking into. Till then using the src package gets us up an running. This environment variable is needed so that pip can skip the wheel and download the tar
```
Skipping wheel build for jpy, due to binaries being disabled for it.
```

Reply:
@mkanoor Yeah, I'm saying that there is no need to deal with the envvar and you can ensure the compilation with the configuration in `setup.cfg` that I mentioned before.

Reply:
👍

Reply:
@Alex-Izquierdo I tried setting the global in setup.cfg and it ended up downloading the wheel
```
Downloading jpy-0.11.0-cp310-cp310-macosx_10_15_universal2.whl (185 kB)
```

Reply:
yeah, I'm doing some tests and doesn't seem to work. :(

Reply:
@Alex-Izquierdo what doesn't work for you?

Reply:
Sorry, it didn't work the configuration that I've proposed: https://github.com/benthomasson/ansible-events/pull/144#issuecomment-1226043523 not the PR